packages/api-server/src/api/portal.ts

```diff
@@ -102,4 +149,48 @@ router.post('/', async (req, res) => {
   }
 })
 
+router.delete('/', async (req, res) => {
+  try {
+    const portalIds = req.body.portals
+      .map((p: number) => {
+        const id = Number(p)
+
+        if (isNaN(id)) {
+          return null
+        }
+
+        return id
+      })
+      .filter(Boolean)
+
+    await deleteServerPortal(portalIds, req.userId, req.serverId)
+    res.send(204)
+  } catch (err) {
+    logger.log.error('Unable to delete', err)
+    res.status(500).send({ error: 'Error deleting portal' })
+  }
+})
+
+// TODO update in later PR
+router.put('/:id(\\d+)', async (req, res) => {
+  try {
+    const body = req.body
+    const expireTime = getExpireTime(body.size, body.hours, body.minutes)
+    const conns = body.connection.sort()
+
+    await updateServerPortal(
+      Number(req.params.id),
```

Comment:
what is wrong with parseInt?

Reply:
Nothing. Number is a little cleaner to write and I like it better.

Reply:
https://thisthat.dev/number-constructor-vs-parse-int/

This article is what turned me on to it.

Reply:
After reading that I am more convinced on parseInt, the NaN cases are just bad for Number()

Reply:
You can check for isNaN

Reply:
It returns 0.